cafebabel/templates/articles/drafts/create.html

```diff
@@ -5,50 +5,59 @@
   <form method=post action={{ url_for('drafts.create') }} enctype=multipart/form-data>
   <canvas></canvas>
   <div class=file>
-    <label for=image>Click here to add a cover picture<br>or<br>Drag & drop your pic here <small>Recommended file size: less than 1Mo</small></label>
+    <label for=image>Click here to add a cover picture or Drag & drop your pic here <small>Recommended file size: less than 1Mo</small></label>
     <input type=file id=image name=image>
   </div>
   <h1 class=text>
     <label for=title>Title</label>
-    <input id=title name=title required placeholder=Title>
+    <input id=title name=title required>
   </h1>
-  <div class=select>
-    <label for=language>Language
-      <select id=language name=language required>
-        {% for language in get_languages() %}
-        <option value={{language[0]}}>{{ language[1] }}</option>
-        {% endfor %}
-      </select>
-    </label>
+  <div class=article-settings>
+    <div class=select>
+      <label for=language>
+        <select id=language name=language required>
+          <optgroup label=Language>
+          {% for language in get_languages() %}
+          <option value={{language[0]}}>{{ language[1] }}</option>
+          {% endfor %}
+        </select>
+      </label>
+    </div>
+    <div class=select>
+      <label for=category>
```

Comment:
Even though we are not making the site highly accessible, I think removing text from the `<label>` will completely remove any hope not to get too far away from it.

Could we eventually wrap the real label into a `<span>` and hide it from the screen in css? (which should still make it browsable for non screen displays). Suggestions or documentation are welcome if you have further info on this.

Reply:
What do you mean bu "wrapping the real label into a `<span>`? like : 
```
<span>
<label for=category>
<select...
</label>
</span>
```
weirdo, isn't it ?

Reply:
What I meant was to have `<label><span>My Label Title</span>…</label>` so that the span could be manipulated in CSS

Reply:
ok sure